### PR TITLE
[INGLONG-3202][SDK] Unify SortSourceConfig of manager and sdk

### LINKED ISSUE #3202: [Feature][SDK] Unify SortSourceConfig of Sort-Sdk and Manager
### Description

Unify SortSourceConfig of Sort-Sdk and Manager

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Title Name: [INGLONG-3202][SDK] Unify SortSourceConfig of manager and sdk

Fixes #3202

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-agent/agent-plugins/AgentBaseTestsHelper/org.apache.inlong.agent.plugin.TestFileAgent/.bdb/je.config.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
...","20000000","3","false","0","false","false","0","5","20","5","1000","false","false","3","2","true","false","","false","true","false","500 ms","8192","0","2","5","5","50","0","5","10","true","1","true","10","0","0","false","10 s","3","500 ms","5 s","false","0","1 ms","0","true","false","100","true","10 seconds","true","true","false","false","false","true","false","true","5 min","true","true","false","false","true","true","true","true","true","true","false","true","5 min","10 s","2 h","5 s","24 h","true","false","true","false","0 0 * * *","true","true","1","0","3","524288","10","false","10 min","true","10","true","4","10","10","true","95","5 s","false","true","0","1","false","10000 MILLISECONDS","1048576","true","0","true","1000 ms","false","2048","100","10000000","10485760","0","5 s","20 s","5 s","500 ms","0 ns","0","16777216","8192","false","0","3","0","false","false","true","false","1048576","0","60","0","128","128","false","1","52428800","10 min","3","5 min","true","false","true","1 min","","1440","10","25","true","true","16","10","16","512000","false","false","null","false","0","","7635730432","536870912","Oracle Corporation","1.8.0_291","7.3.7","x86_64","10","Mac OS X","10.16"
```
Comment (luchunliang) [inline inlong-agent/agent-plugins/AgentBaseTestsHelper/org.apache.inlong.agent.plugin.TestFileAgent/.bdb/je.config.csv:2]: invalid code file

File: inlong-agent/agent-plugins/AgentBaseTestsHelper/org.apache.inlong.agent.plugin.TestFileAgent/20220318 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+This is the test line for huge file
+This is the test line for huge file
```
Comment (luchunliang) [inline inlong-agent/agent-plugins/AgentBaseTestsHelper/org.apache.inlong.agent.plugin.TestFileAgent/20220318:2]: ditto

File: inlong-agent/agent-plugins/AgentBaseTestsHelper/org.apache.inlong.agent.plugin.TestFileAgent/.bdb/je.stat.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+2022-03-18 03:42:21.609 UTC, ,125,3158477,824,12624, , ,3, ,3, , , , , , , , , ,3,3, , , , ,1,5, , , , , , , , , , , , , , , , ,1077,1,469,469,1, ,2,4, ,49,49, , , , , ,1,10, , , , , , , , , , , , , , , , , , , , , , , , ,1647574941383, , , , ,3145728, , , , ,1,3,2,3,3,3, , ,3,4, , , , , , , ,38,1, , ,12288,3,1087,3, , , , , , ,1,17,48837048,16, , , , , , ,5, , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , , ,
```
Comment (luchunliang) [inline inlong-agent/agent-plugins/AgentBaseTestsHelper/org.apache.inlong.agent.plugin.TestFileAgent/.bdb/je.stat.csv:2]: invalid code file

**Review by luchunliang — APPROVED** [review]

**Review by wardlican — APPROVED** [review]

**Comment by healchow** [comment]
Fixed by https://github.com/apache/incubator-inlong/pull/3215

**Pull request closed without merging**